course_dashboard/stats.py

```diff
@@ -157,10 +169,11 @@ def most_active_username(threads):
 class EnrollmentStats(object):
     """Provide enrollments stats for a given course."""
 
-    def __init__(self, course_id=None, since=None):
+    def __init__(self, course_id=None, since=None, active=True):
         self.course_id = course_id
         self.since = since
-        self.per_date = enrollments_per_day(self.course_id, since=since)
+        self.per_date = enrollments_per_day(self.course_id, since=since, active=active)
+        self.inactive_per_date = enrollments_per_day(self.course_id, since=since, active=False)
```

Comment:
Il y a un problème ici ; si les premiers (resp.: derniers) jours ne contiennent aucune inscription (ou désinscription) alors les listes per_date et inactive_per_date ne commenceront (resp.: finiront) pas à la même date. Les tableaux d'inscriptions/désinscriptions ne seront pas alignés. Pour résoudre ce problème je suggère de remplir les listes de zéro depuis `since` jusqu'à `today`.